README.md

```diff
@@ -113,22 +113,30 @@ If you want to develop for this project or verify your configuration quickly wit
 
 1. Clone the repo.
 
-2. Create data and wallet folders on the host machine and import validator accounts.
+2. Create the data folders for beacon node and validator and change the ownership. For example:
```

Comment:
As mentioned in the other comment: we should change the ownership after we import the validator account.